src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/host-form/host-form.component.html

```diff
@@ -1,7 +1,6 @@
 <cds-modal size="md"
            [open]="open"
-           [hasScrollingContent]="true"
-           (overlaySelected)="closeModal()">
```

Comment:
removing this so that modal won't close when you click outside the modal body. Its okay for a normal modal but when the modal is routed, its not expected to close on the outside click.